bigquery/samples/tests/test_client_query.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .. import client_query
+
+
+def test_client_query(capsys, client):
+
+    client_query.client_query(client)
+    out, err = capsys.readouterr()
+    assert "The query data:" in out
+    assert "Row(" in out
```

Comment:
Since we're using the `usa_1910_2013` table, the data won't change. We can use a specific value in our tests.

```suggestion
    assert "name=James, count=272793" in out
```